js/adapt-contrib-resources.js

```diff
@@ -23,21 +23,24 @@ define(function(require) {
 
 		var courseResources = Adapt.course.get('_resources');
 
-		if (courseResources) {
-			var drawerObject = {
-		        title: courseResources.title,
-		        description: courseResources.description,
-		        className: 'resources-drawer'
-		    };
-		    // Syntax for adding a Drawer item
-		    // Adapt.drawer.addItem([object], [callbackEvent]);
-		    Adapt.drawer.addItem(drawerObject, 'resources:showResources');
-		} else {
-			return console.log('Sorry, no resources object is set on the course.json file');
-		}
+        // do not proceed until resource set on course.json
+        if (!courseResources) {
+            return console.log('Sorry, no resources object is set on the course.json file');
+        } else if (courseResources._isEnabled !== undefined && courseResources._isEnabled !== true) {
```

Comment:
I think it would be better if resources were only disabled if `_isEnabled === false`

The indentation here needs sorting out a little bit